llvm/lib/Object/ArchiveWriter.cpp

```diff
@@ -677,6 +677,16 @@ static bool isECObject(object::SymbolicFile &Obj) {
   return false;
 }
 
+bool isImportDescriptor(const std::string_view Name) {
+  return Name.substr(0, ImportDescriptorPrefix.size()) ==
+             ImportDescriptorPrefix ||
```

Comment:
llvm::StringRef, the LLVM equivalent to string_view, is used widely all over LLVM, has various convenient methods here, like starts_with()/ends_with().  I don't think it's worth switching to the std version at the cost of readability.  (Maybe LLVM will revisit the general usage of StringRef at some point, but likely not soon.)

Reply:
Good call